pandas/tests/extension/base/methods.py

```diff
@@ -358,3 +358,18 @@ def test_repeat_raises(self, data, repeats, kwargs, error, msg, use_numpy):
                 np.repeat(data, repeats, **kwargs)
             else:
                 data.repeat(repeats, **kwargs)
+
+    def test_equals(self, data, na_value):
+        cls = type(data)
+        ser = pd.Series(cls._from_sequence(data, dtype=data.dtype))
+        na_ser = pd.Series(cls._from_sequence([na_value], dtype=data.dtype))
+
+        assert data.equals(data)
```

Comment:
make all these `assert <condition> is True`, to ensure that `True` or `False` is actually returned

Reply:
I ran into a funny issue doing this: https://github.com/numpy/numpy/issues/5942
I'm going to keep this as is now to keep the test consistent within itself.

`.equals()` returns a `np.bool_` type. As result when you compare it to bool type `True` using `is` (i.e. `some_np_bool_True_val is True == False` 

Coercing `np.bool_` to `bool` using the `bool()` constructor did not work either. `== True` causes PEP8 errors asking me to use `assert <condition>` or `assert <condition> is True`.

Open to suggestions if you have any.